sig-docs/annual-report-2021.md

```diff
@@ -0,0 +1,113 @@
+Are OWNERS files up to date in these areas?**
+
+Yes it is.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+<!-- TODO -->
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes, they are.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We don't have an official way of measuring membership.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+PR are generally open for anyone to review. We have [PR wrangler rotation](https://github.com/kubernetes/website/wiki/PR-Wranglers).
+In general each of the approver will taking the rotation. Under exceptional circumstances where there's lots of
+pressure for the wrangler, another approver usually will be willing to help.
+
+For technical reviews, other than SIG Docs Tech Leads, we also notify respective SIGs with necessary knowledge of the
+technical aspect of the content.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? 
+What are some activities that the group does to encourage this? 
+What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+There is no official onboarding process. However, the SIG is supportive of new contributors,
+willing to walk through the contribution process and eager to see contributions completed.
+Issues are created regularly and triaged. Issues are also created specifically for new 
+contributors. There are issues labelled for the different contribution areas (web development, 
+blog, first time contributors, task, concept, API, etc) which helps new contributors 
+identify how to get started.
+
+**What programs do you participate in for new contributors?**
+
+We have an in person session during the KubeCon.
+
+**Does the group have contributors from multiple companies/affiliations? 
+Can end users/companies contribute in some way that they currently are not?**
+
+<!-- TODO -->
+
+## Current initiatives and project health
+
+[ ] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members.
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? 
+Currently underway? What are some of the longer tail projects that your group is working on?**
+
+<!-- TODO -->
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+We have ongoing KEP, but we don't have a deadline set on when it should be finished.
+
+- [Doc policies for third party content](https://github.com/kubernetes/enhancements/tree/master/keps/sig-docs/1326-third-party-content-in-docs)
+
+**What areas and/or subprojects does the group need the most help with?**
+
+<!-- TODO -->
```

Comment:
My gut feel is that the blog project could use extra help.

Reply:
Yes, could add the blog subproject.